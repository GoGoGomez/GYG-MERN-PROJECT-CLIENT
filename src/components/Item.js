import React from 'react';

const Item = (props) => (
  <div>
    <img src={props.imagePath} alt={props.title}/>
    <p>{props.title}</p>
  </div>
);

export default Item;