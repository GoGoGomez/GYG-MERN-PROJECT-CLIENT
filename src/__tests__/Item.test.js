import React from "react";
import renderer from "react-test-renderer";
import Item from "../components/Item";

test("Item renders correctly", () => {
  const component = renderer.create(<Item />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
