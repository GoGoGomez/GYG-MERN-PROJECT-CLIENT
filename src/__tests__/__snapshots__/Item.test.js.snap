// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item renders correctly 1`] = `
<div>
  <img
    alt={undefined}
    src={undefined}
  />
  <h3 />
</div>
`;
